src/bundle.rs

```diff
@@ -67,6 +67,7 @@ pub unsafe trait DynamicBundleClone: DynamicBundle {
 }
 
 #[derive(Copy, Clone)]
+#[doc(hidden)]
```

Comment:
This must be applied to the reexport in lib.rs, not the original definition. That reexport should also be moved down into the dedicated block of reexports for the macro crate.